base/src/main/java/io/spine/code/proto/MessageType.java

```diff
@@ -64,6 +65,15 @@ public MessageType(Descriptor descriptor) {
         super(descriptor, true);
     }
 
+    /**
+     * Creates a new instance from the given {@code Message} class.
+     */
+    public static MessageType of(Class<? extends Message> aClass) {
```

Comment:
Please have a constructor instead. We have types that derive from `MessageType`. Such a static method may be a dis-service.